pkg/cloudclient/gcp/gcp_test.go

```diff
@@ -1,5 +1,6 @@
 package gcp
 
+/*
```

Comment:
Why are you commenting out all the tests ?

Reply:
Methods ValidateEgress and NewClient send request to the gcp api. We are having issues with using mockgen https://issues.redhat.com/browse/OSD-12645. Thanks Benjamin for the suggestion, we have uncommmented VPC test case and skipped ValidateEgress and NewClient due to request being sent to gcp api.